proto/launch/tx.proto

```diff
@@ -36,6 +36,7 @@ message MsgRequestAddValidator {
 
 message MsgRequestAddValidatorResponse {
   uint64 requestID = 1;
+  bool autoApproved = 2;
```

Comment:
Haven't thought about it.
That can actually be useful if we add later different logic for auto approval like allowances

Reply:
the reason is that when we auto-approve the request, we don't have the request-id. I think it is a good way to show the request is already approved